collections.md

```diff
@@ -444,6 +449,124 @@ Return codes:
+
+#### Collection administrators
+**/api/core/collections/<:uuid>/adminGroup**
+
+Endpoints for managing the collection administrators
+
+##### Retrieve collection administrators
+**GET /api/core/collections/<:uuid>/adminGroup**
+
+Example: /server/api/core/collections/7669c72a-3f2a-451f-a3b9-9210e7a4c02f/adminGroup
+
+It returns the EPerson Group representing the administrators of this collection. [See the EPerson Group endpoint for more info](epersongroups.md#single-eperson-group)
+
+Return codes:
+* 200 OK - if the admin group exists and is returned
+* 204 No content - if the current collection exists but the admin group doesn't exist
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions to retrieve the groups, parent community admins and collection admins can retrieve the group
+* 404 Not found - if the current collection doesn't exist
+
+##### Create collection administrators group
+**POST /api/core/collections/<:uuid>/adminGroup**
+
+To be used on a collection without collection administrators
+
+Perform a post with the JSON below.
+
+```json
+{
+  "metadata": {
+      "dc.description": [
+        {
+          "value": "Test group",
+          "language": null,
+          "authority": "",
+          "confidence": -1
+        }
+      ]
+  }
+}
+```
+Contrary to the [EPerson Group endpoint](epersongroups.md#create-new-eperson-group), the name cannot be set here
+
+Status codes:
+* 201 Created - if the operation succeed
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions. Only admins and parent community admins can retrieve the group
+* 404 Not found - if the collection doesn't exist
+* 422 Unprocessable Entity - if the name was included, if permanent was set to true, or if the collection already contains an administrator group
+
+##### Modifying the collection administrators group
+
+All modifications to the group will be performed directly on the Group endpoint:
+* Adding members will need to use the [EPerson Group endpoint](epersongroups.md#add-an-eperson-to-a-parent-group)
+* Adding sub-groups will need to use the [EPerson Group endpoint](epersongroups.md#add-a-group-to-a-parent-group)
+
+Modifying the collection administrators group will be authorized for admins, parent community admins and collection admins
+
+##### Delete the collection administrators group
+**DELETE /api/core/collections/<:uuid>/adminGroup**
+
+To be used on a collection with an administrator group
+
+Status codes:
+* 204 No content - if the operation succeed
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions. Only admins, parent community admins and collection admins can delete the group
+* 404 Not found - if the collection doesn't exist
+* 422: if the collection didn't contain an administrator group
```

Comment:
In other endpoint we return 204 No content also in the case of  no-op (see items for instance)

Reply:
Thanks @abollini 
I've adjusted this to be inline with the item DELETE